ignite-worker/src/main/resources/OSGI-INF/blueprint/horizon-ignite.xml

```diff
@@ -8,6 +8,7 @@
     <cm:default-properties>
       <cm:property name="useKubernetes" value="false"/>
       <cm:property name="kubernetes.service-name" value="poc-distributed-scheduling"/>
+      <cm:property name="workflowSchedulerThreadCount" value="20"/>
```

Comment:
Suggest: workflow.scheduler.threadcount

Reply:
I can change it.  Was thinking of using ENV-var-compatible naming to make the naming consistent when we use ENV vars with kubernetes, etc.

Thoughts?

Reply:
Seems like we always use "dot" notation in the properties files. I do find it a bit easier to parse for some reason

Reply:
Agreed - but of course ENV variable names don't allow dots or dashes.  So if we start using K8S + ENV variables for all settings, we'll have 2 names for every setting - the internal property name and the ENV variable name.

Least-common-denominator FTW (sadly)...